theme/padplus/scss/post/components/_badges.scss

```diff
@@ -16,7 +17,29 @@
     }
 
     &.badge-pad-info {
+        align-items: center;
         background-color: map-get($badge-colors, "info");
+        display: flex;
+        width: fit-content;
+    }
+}
+
+.badge-success {
```

Comment:
regression? &.badge-pad-workshop is not linked to badge-pad but to badge-success

Reply:
javier